src/config/types.ts

```diff
@@ -249,7 +249,7 @@ type ReadinessProbeObj = {
 
 type ReadinessProbe = ReadinessProbeFn | ReadinessProbeObj;
 
-export enum RecordMode {
```

Comment:
Wouldn't it break html-reporter?

Reply:
No, we use safe imports there, so it wouldn't crash. It would just consider that time travel is not available (user is using old testplane version).

But I will test that, just to be sure.